apps/condo/pages/meter/unit/[id]/index.tsx

```diff
@@ -0,0 +1,81 @@
+import get from 'lodash/get'
+import Head from 'next/head'
+import { useRouter } from 'next/router'
+import React, { useMemo } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+
+import { PageContent, PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
+import LoadingOrErrorPage from '@condo/domains/common/components/containers/LoadingOrErrorPage'
+import { MultipleFilterContextProvider } from '@condo/domains/common/hooks/useMultipleFiltersModal'
+import { MeterPageContent } from '@condo/domains/meter/components/Meters/MeterPageContent'
+import { Meter, MeterReportingPeriod, MeterResource, METER_TAB_TYPES } from '@condo/domains/meter/utils/clientSchema'
+import { getMeterTitleMessage } from '@condo/domains/meter/utils/helpers'
+
+
+const MeterInfoPage = (): JSX.Element => {
+    const intl = useIntl()
+    const ServerErrorMessage = intl.formatMessage({ id: 'ServerError' })
+
+    
+    const { query: { id: meterId } } = useRouter()
+    const {
+        obj: meter,
+        error: meterError,
+        loading: isMeterLoading,
+        refetch,
+    } = Meter.useObject({ where: { id: String(meterId) } })
+
+    const organizationId = get(meter, 'organization.id', null)
+    const propertyId = get(meter, 'property.id', null)
+    const resourceId = get(meter, 'resource.id', null)
+
+    const {
+        objs: possibleReportingPeriods,
+        loading: isPeriodsLoading,
+    } = MeterReportingPeriod.useObjects({ where: {
+        OR: [
+            { AND: [ { organization: { id: organizationId } }, { property: { id: propertyId } }, { deletedAt: null } ] },
```

Comment:
Same about deletedAt on client side.
We have a `softDeleted` schema plugin, which filters entities with deletedAt.
Or there are a specific case?

Reply:
there isn't a specific case, my mistake